ufl/finiteelement/finiteelementbase.py

```diff
@@ -213,3 +213,7 @@ def __getitem__(self, index):
             from ufl.finiteelement import RestrictedElement
             return RestrictedElement(self, index)
         return NotImplemented
+
+    def is_fully_continuous(self):
+        """Return true if the values of this element's basis functions are continuous between elements."""
+        raise NotImplementedError()
```

Comment:
Why not implement the method on this base class and then the only special case is for mixed element?

Reply:
The base class doesn't have a sobolev space, so it didn't look like `self in H1` would work in the base class

Reply:
The base class doesn't need to have a Sobolev space. Python instance methods are called with the concrete object as `self`, so the object that it gets called on ought to have one.